library/src/main/java/com/github/pwittchen/reactivewifi/WifiState.java

```diff
@@ -0,0 +1,52 @@
+package com.github.pwittchen.reactivewifi;
+
+/**
+ * Created by bhavdip on 2/9/17.
+ */
+
+public enum WifiState {
+  WIFI_STATE_DISABLING(0, "disabling"),
```

Comment:
I prefer using predefined constants from Android API instead of numbers here. E.g. WifiManager.WIFI_STATE_DISABLING and so on. Such approach is safe in the case when Android API will change.